### PR TITLE
app-shells/mpv-bash-completion: verbump to 3.3.17

### PR DESCRIPTION
**Pull request by Coacher**
Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

File: app-shells/mpv-bash-completion/mpv-bash-completion-3.3.17.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+RDEPEND="${COMMON_DEPEND}
+	>=app-shells/bash-completion-2.3-r1
+"
+DEPEND="${COMMON_DEPEND}
+	!luajit? ( dev-lang/lua:* )
+	luajit? ( dev-lang/luajit:2 )
+"
+
+src_prepare() {
+	default_src_prepare
```
Comment (Zlogene) [inline app-shells/mpv-bash-completion/mpv-bash-completion-3.3.17.ebuild]: just `default`

File: app-shells/mpv-bash-completion/mpv-bash-completion-3.3.17.ebuild [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	# Avoid 'mpv' make target that supports lua only.
+	sed -i -e 's|check: mpv|check:|' Makefile || die
+}
+
+src_compile() {
+	# Prevent access violations from completion generation.
+	# See Gentoo bug 656086.
+	addpredict /dev/dri
```
Comment (mgorny) [inline app-shells/mpv-bash-completion/mpv-bash-completion-3.3.17.ebuild]: Any chance to fix it upstream not to access this node? `addpredict` is only a cheap workaround.

Comment (Coacher) [reply app-shells/mpv-bash-completion/mpv-bash-completion-3.3.17.ebuild]: It's actually a problem in mesa. See the mentioned Gentoo bug and https://bugs.freedesktop.org/show_bug.cgi?id=106572

**Comment by Coacher** [comment]
Thank you, @mgorny.

**Pull request closed without merging**